src/test/java/com/jcabi/github/RtBranchesTest.java

```diff
@@ -0,0 +1,156 @@
+    /**
+     * Name of test repository.
+     */
+    private static final String REPO_NAME = "epicness";
+
+    /**
+     * RtBranches should be able to iterate over all branches.
+     * @throws Exception if there is any error
+     */
+    @Test
+    public void iteratesOverBranches() throws Exception {
+        final String firstname = "first";
+        final String firstsha = "a971b1aca044105897297b87b0b0983a54dd5817";
+        final String secondname = "second";
+        final String secondsha = "5d8dc2acf9c95d0d4e8881eebe04c2f0cbb249ff";
+        final MkAnswer answer = new MkAnswer.Simple(
+            HttpURLConnection.HTTP_OK,
+            Json.createArrayBuilder()
+                .add(branch(firstname, firstsha))
+                .add(branch(secondname, secondsha))
+                .build().toString()
+        );
+        final MkContainer container = new MkGrizzlyContainer()
+            .next(answer)
+            .next(answer)
+            .start();
+        final RtBranches branches = new RtBranches(
+            new JdkRequest(container.home()),
+            repository()
+        );
+        MatcherAssert.assertThat(
+            branches.iterate(),
+            Matchers.<Branch>iterableWithSize(2)
+        );
+        final Iterator<Branch> iter = branches.iterate().iterator();
+        final Branch first = iter.next();
+        MatcherAssert.assertThat(first.name(), Matchers.equalTo(firstname));
+        MatcherAssert.assertThat(
+            first.commit().sha(),
+            Matchers.equalTo(firstsha)
+        );
+        final Branch second = iter.next();
+        MatcherAssert.assertThat(second.name(), Matchers.equalTo(secondname));
+        MatcherAssert.assertThat(
+            second.commit().sha(),
+            Matchers.equalTo(secondsha)
+        );
+        container.stop();
+    }
+
+    /**
+     * RtBranches should be able to fetch its repository.
+     * @throws IOException If there is any I/O problem
+     */
+    @Test
+    public void fetchesRepo() throws IOException {
+        final Repo repo = RtBranchesTest.repository();
+        final RtBranches branch = new RtBranches(new FakeRequest(), repo);
+        final Coordinates coords = branch.repo().coordinates();
+        MatcherAssert.assertThat(coords.user(), Matchers.equalTo(REPO_USER));
+        MatcherAssert.assertThat(coords.repo(), Matchers.equalTo(REPO_NAME));
+    }
+
+    /**
+     * Create and return JsonObject to test.
+     * @param name Name of the branch
+     * @param sha Commit SHA of the branch
+     * @return JsonObject
+     * @throws Exception If some problem inside
+     */
+    private static JsonObject branch(final String name, final String sha)
+        throws Exception {
+        return Json.createObjectBuilder()
+            .add(NAME_PARAM, name)
+            .add(
+                "commit",
+                Json.createObjectBuilder()
+                    .add("sha", sha)
+                    // @checkstyle LineLengthCheck (1 line)
```

Comment:
@ggajos Even if we put the `String.format` on its own line, that line will still be too long.